torch/csrc/jit/codegen/cuda/code_write.cpp

```diff
@@ -427,19 +427,23 @@ void CodeWrite::header() {
   for (Val* val : vals) {
     switch (val->getValType().value()) {
       case (ValType::TensorView):
+        {
         switch (val->getDataType().value()) {
           case (DataType::Float):
-            os << "Tensor<float> T";
```

Comment:
I think I moved this to ir_iostream. I will rebase on code_lowering and fix this.